esmvalcore/cmor/_fixes/cmip6/ec_earth3.py

```diff
@@ -0,0 +1,26 @@
+"""Fixes for EC-Earth3 model."""
+from ..fix import Fix
+
+
+class siconca(Fix):
+    """Fixes for siconca."""
+
+    def fix_data(self, cube):
+        """
+        Fix data.
+
+        Fixes discrepancy between declared units and real units
+
+        Parameters
+        ----------
+        cube: iris.cube.Cube
+
+        Returns
+        -------
+        iris.cube.Cube
+
+        """
+        metadata = cube.metadata
+        cube *= 100
+        cube.metadata = metadata
```

Comment:
```suggestion
        cube.data = cube.core_data() * 100.
```
would be recommended, as that surely only changes the required data.